lib/experimental/split.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Thin JS interop wrapper around https://split.js.org/
```

Comment:
Can we work with @devoncarew's team to migrate this file into the split package? It makes sense to me to have this matched with the JS library it is wrapping.

Reply:
I sure we'd accept PRs into https://github.com/flutter/devtools/tree/master/third_party/packages to do that -

Reply:
sounds good, PR [here](https://github.com/flutter/devtools/pull/572)

Reply:
discussed offline with Andrew: we're going to merge as-is and incorporate the new version of the split package later